CIP-0139/README.md

```diff
@@ -0,0 +1,141 @@
+---
+CIP: 139
+Title: Universal Query Layer
+Category: Tools
+Status: Proposed
+Authors:
+    - Vladimir Kalnitsky <klntsky@gmail.com>
+    - Giovanni Garufi <giovanni@mlabs.city>
+Implementors: []
+Discussions:
+    - https://discord.gg/MU8vHAgmGy
+Created: 2024-05-14
+License: CC-BY-4.0
+Version: 0.0.0
```

Comment:
```suggestion
```
Could cause a parsing problem; as mentioned in https://github.com/cardano-foundation/CIPs/pull/957#discussion_r1907480369 could also be beneficial, but would take some coordination between CIP editors and CIP content rendering sites to be sure this tag wouldn't stop those web sites from updating properly.